editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -9349,7 +9370,7 @@ Node3DEditor::Node3DEditor() {
 
 	gizmos_menu = memnew(PopupMenu);
 	gizmos_menu->set_hide_on_checkable_item_selection(false);
-	p->add_submenu_node_item(TTR("Gizmos"), gizmos_menu);
+	p->add_submenu_node_item(TTRC("Gizmos"), gizmos_menu);
```

Comment:
I think maybe gizmo plugin's names in this submenu should not be translated (but the item tooltips should be).